docs/preact-testing-library/faq.md

```diff
@@ -0,0 +1,220 @@
+---
+id: faq
+title: FAQ
+---
+
+See also the [main FAQ](/docs/faq) for questions not specific to React testing
```

Comment:
This should refer to the React FAQ if it doesn't have anything Preact-specific in it